### ISSUE TITLE
New release ?

### ISSUE THREAD

**Issue by nlamirault** [proofit404/anaconda-mode#98]
Last release is from 02/2014.

**Comment by proofit404** [comment]
I'll make release after finishing `feature/pypi` branch. Corresponding `company` and `ac` will follow.

**Comment by stardiviner** [comment]
awesome.

**Comment by nlamirault** [comment]
Great

**Comment by proofit404** [comment]
@nlamirault @stardiviner `pypi` branch was merged into `master`. I'll wait few days to see it doesn't cause troubles to anyone. If no major issues will be reported then I'll ship new release.

Thanks for you patience.

**Comment by nlamirault** [comment]
Thanks

**Comment by proofit404** [comment]
Blocked by #104 and #105

**Comment by proofit404** [comment]
Blocked by #110

**Comment by proofit404** [comment]
Done.

**Comment by nlamirault** [comment]
Yes

**Comment by proofit404** [comment]
You're welcome :smile:
